test/e2e/page-objects/hearing/notAttendingHearing.js

```diff
@@ -0,0 +1,8 @@
+function continueFromnotAttendingHearing() {
```

Comment:
Just a note for future reference: we are starting to accumulate a number of these where all we do is continue. Ideally we want a generic approach where we have something like `continueToNextStep` and we use that throughout.